### PR TITLE
Less noisy base config

### PR DESCRIPTION
**Pull request by sapegin**
Do not print long list of chunks etc. on build.

### CODE REVIEW

**Comment by andywer** [comment]
The [webpack documentation](https://webpack.js.org/configuration/stats/) says "for webpack-dev-server, this property needs to be in the `devServer` object".

So I suppose we have to add this to the dev-server block code as well?

**Comment by sapegin** [comment]
Yup, that would be my next PR ;-)

**Comment by andywer** [comment]
But wouldn't it make sense to do both fixes in one PR (one "less noise" PR)?

**Comment by sapegin** [comment]
I also wanted to enable error overlay and compression for devServer. Can include that as well, or send as another PR.

**Comment by sapegin** [comment]
And this as well: https://github.com/facebookincubator/create-react-app/blob/2069e8ac43c4765f1698091d3985ecb70ba88089/packages/react-scripts/config/webpackDevServer.config.js#L75-L79

**Comment by andywer** [comment]
Proposal: I would rather not put `node_modules` exclusion and compression in the dev-server as an implicit default. The error overlay (you mean the red error screen?) maybe, maybe not. But since you don't expect webpack-blocks to silently change your config I would not do so, unless it's extremely generic and extra-unlikely to ever cause any issues.

**Comment by andywer** [comment]
But +1 for the proposed changes. Just not a big fan of letting very generic blocks change detail configuration, since that already caused multiple issues in the project's past.

**Comment by sapegin** [comment]
I think you see webpack-blocks more low lever than it should be to be really useful ;-)

> I would rather not put node_modules exclusion

Why not? People don’t change code inside node_modules often.

> and compression in the dev-server as an implicit default.

What downsides you see here? All modern browsers support compression and it would make it a little bit faster. I think it would be a good default for most users.

> The error overlay (you mean the red error screen?) maybe, maybe not.

Yes, it makes DX much better — you don’t see an app that looks normal but wasn’t even compiled ;-)

> But since you don't expect webpack-blocks to silently change your config I would not do so, unless it's extremely generic and extra-unlikely to ever cause any issues.

I think this is the place where our visions differ the most ;-) That’s exactly what I want webpack-blocks to do and why I use it! And there’s no my config, only config that was generated by webpack-blocks. I think we shouldn’t expect webpack-blocks to know all webpack defaults but should try to find our own default that would make the best experience for most of them. Similar to Create React App but we’re in a better position here: our users could easily change defaults ;-)

**Comment by andywer** [comment]
Maybe it is 😉

`node_modules/` exclusion:
Just cautious, since it might cause issues for people who are used to do quick hacks in their node_modules and now wonder why it won't update anymore. Otherwise fine to me.

Compression:
Yeah, compression will probably be fine, ignore my comment :)

Error screen:
There is just one thing about it that might annoy people (thus would make it a rather personal thing). When developing with HMR if you change the code and it breaks it will replace your currently running site with the error screen. Without it you are able to use the site normally, just see the error in the console and everything will resume normally as soon as you fix the error.

Don't get me wrong, I like your practical view on usage! It's just that I became quite cautious about defaults in generic blocks. Providing higher level abstractions on top of low-level stuff is easy, changing low-level things that annoy you when all you have got is high-level blocks, is not.

**Comment by sapegin** [comment]
>  it might cause issues for people who are used to

This could be said about any non-default value we have ;-) I think we should document default config for each block and it would solve this and some other issues (I have to read blocks’ sources almost every time now). But anyway with normal webpack I had to read the docs every time because I think people are used to how *their* app behave and not to defaults.

> Without it you are able to use the site normally

This is probably too opinionated to decide between two of us — we need more people here. On your side is the webpack default value, on my — CRA ;-)

**Comment by sapegin** [comment]
@andywer I think it’s ready to be merged ;-) Everything else is out of the scope of this PR anyway.

**Comment by andywer** [comment]
How about asking the twitter followers what they'd prefer? The users don't necessarily know what they want, but should be ok to make an educated decision here.

**Comment by sapegin** [comment]
@andywer Do it and I’ll retweet ;-)

File: packages/dev-server/index.js [hunk 1]
```diff
@@ -47,7 +47,11 @@ function postConfig (context, util) {
         hot: true,
         hotOnly: true,
         historyApiFallback: true,
-        inline: true
+        inline: true,
+        // Disable verbose loggin in browser’s console
+        clientLogLevel: 'none',
```
Comment (andywer) [inline packages/dev-server/index.js]: I'd rather prefer `warn` or at least `error` here. I don't always look at the console running webpack. The on-page console is open pretty much all-the-time instead.

Comment (sapegin) [inline packages/dev-server/index.js]: > Note that the console will always show bundle errors and warnings. This option only effects the message before it.

https://webpack.js.org/configuration/dev-server/#devserver-clientloglevel

Comment (andywer) [inline packages/dev-server/index.js]: What do they mean by "the message before it"? 🤔

Comment (sapegin) [inline packages/dev-server/index.js]: I’ve experimented a bit. This paragraph [is misleading](https://github.com/webpack/webpack-dev-server/issues/1038). Changing it to `error`.

Comment (andywer) [inline packages/dev-server/index.js]: 👍👍

**Comment by andywer** [comment]
Just one more thing (sorry!): Could you please update the changelogs?

You can just use "Next version" as version title.

**Pull request merged**